pkg/api/handlers/compat/info.go

```diff
@@ -133,6 +134,37 @@ func GetInfo(w http.ResponseWriter, r *http.Request) {
 	utils.WriteResponse(w, http.StatusOK, info)
 }
 
+func getServiceConfig(runtime *libpod.Runtime) *registry.ServiceConfig {
+	var indexConfs map[string]*registry.IndexInfo
+
+	regs, err := sysregistriesv2.GetRegistries(runtime.SystemContext())
+	if err == nil {
+		indexConfs = make(map[string]*registry.IndexInfo, len(regs))
+		for _, reg := range regs {
+			mirrors := make([]string, len(reg.Mirrors))
+			for i, mirror := range reg.Mirrors {
+				mirrors[i] = mirror.Location
+			}
+			indexConfs[reg.Prefix] = &registry.IndexInfo{
+				Name:    reg.Prefix,
+				Mirrors: mirrors,
+				Secure:  !reg.Insecure,
+			}
+		}
+	} else {
+		log.Warn("failed to get registries configuration")
```

Comment:
I think this should be an error. If the server cannot read registries.conf files, then something's seriously wrong.